frontend/src/components/Footer.tsx

```diff
@@ -75,23 +74,26 @@ const Footer = () => {
     <USWDSFooter
       data-testid="footer"
       size="medium"
-      primary={<div />}
+      returnToTop={
+        <div className="grid-container usa-footer__return-to-top">
+          <a href="#">{t("return_to_top")}</a>
+        </div>
+      }
+      primary={null}
       secondary={
-        <Grid row>
-          <Logo
-            size="medium"
-            image={
-              <img
-                className="usa-footer__logo-img"
-                alt={t("logo_alt")}
-                src={"/img/grants-gov-logo.png"}
-              />
-            }
-            heading={
-              <p className="usa-footer__logo-heading">{t("agency_name")}</p>
-            }
-          />
-          <Grid className="usa-footer__contact-links" mobileLg={{ col: 6 }}>
+        <Grid row gap>
+          <Grid tablet={{ col: 4 }} desktop={{ col: 6 }}>
+            <img
+              className="maxh-15 margin-bottom-2 tablet:margin-bottom-0"
+              alt={t("logo_alt")}
+              src={"/img/grants-gov-logo.png"}
+            />
+          </Grid>
```

Comment:
Putting the logo image in a `Grid` that we can control the column count on. Also using a max height on the logo image instead of the max width the `.usa-footer__logo-img` class awkwardly applies. This lets us make the logo narrower at smaller screen sizes. Also nixing the heading and relying on the alt text for a11y.